### DISCUSSION TITLE
Download location of September 2022 Buster Raspios (hopefully)

### DISCUSSION THREAD

**Discussion by dwalls70** [Short-bus/pilomar#48] [General]
The hard to find September 2022 Buster image is here: https://downloads.raspberrypi.org/raspios_oldstable_armhf/images/raspios_oldstable_armhf-2022-09-26/

note that I have not downloaded it to make sure it is the desktop version and not just cli

**Comment by Short-bus** [comment]
Thanks Dave, I'll check it out too.

Currently investigating a build for Bookworm 64bit too (on an RPi5).
- The O/S build and package installation looks much simpler, Bookworm seems to now come with more of the packages that had to be manually added in Buster.
- But the Python code needs updating. libcamera, UART, GPIO changes and at least one weird bug in the time.sleep() function. etc.

Matt

**Comment by Short-bus** [comment]
Just returning to this, as per the recent update, the latest release of the project works on Bookworm 64bit on the RPi4B.
I've also heard that there are two flavours of the RPi4B out there. 1.5GHz and 1.8GHz.
I think it's the same CPU but recent releases may be overclocking to 1.8GHz now?
The 1.8GHz overclocking sounds like it needs Bookworm 64bit anyway.

This archived copy of Buster is still useful if you're running on a RPi3B, I think you can't run Bookworm on those?

RPi5B support is not ready yet.
